data/core/units/monsters/Roc.cfg

```diff
@@ -17,7 +17,7 @@
     {AMLA_DEFAULT}
     cost=70
     usage=scout
-    description= _ "In remote desert mountains, far from any city or settlement, enormous raptors are rumoured to nest in the lofty crags.  Travellers and nomads claim to have witnessed livestock and grown men snatched into the sky without warning by these monsters.  While one should not place much faith in the accounts of the desert roamers, they are less fantastical than the drunken lies of sailors who claim to have witnessed full-rigged ships, attempting to navigate the narrow straights between the sheer cliffs where nothing can grow, sunk by giant eagles dropping large rocks.
+    description= _ "In remote desert mountains, far from any city or settlement, enormous raptors are rumoured to nest in the lofty crags.  Travellers and nomads claim to have witnessed livestock and grown men snatched into the sky without warning by these monsters.  While one should not place much faith in the accounts of the desert roamers, they are less fantastical than the drunken lies of sailors who claim to have witnessed full-rigged ships, attempting to navigate the narrow straits between the sheer cliffs where nothing can grow, sunk by giant eagles dropping large rocks.
```

Comment:
Still some double-spaces to catch - there's one before Travellers. Also, Firefox says there should only be one L in the `en_US` spelling of Travelers.

Reply:
Already covered in #7965. This PR is strictly for spelling/punctuation changes - Pentarctagon suggested we could push those in before releasing for translation.

It's never been my aim to 'correct' for non-en_US spellings in the source text - I leave that for native en_US readers. I do notice these, though - I think it was in the review for WoF I recall some vacillation between en_US and non-en_US spellings, for example.